packages/react-native-editor/__device-tests__/gutenberg-editor-rendering-media-blocks.test.js

```diff
@@ -8,7 +8,7 @@ describe( 'Gutenberg Editor Rendering Media Blocks test', () => {
 		await editorPage.initializeEditor( { initialData: mediaBlocks } );
 
 		// Give some time to media placeholders to render.
-		await editorPage.driver.sleep( 3000 );
+		await editorPage.driver.setTimeout( { implicit: 3000 } );
```

Comment:
After this change, I saw there's a `pause` method ([docs](https://webdriver.io/docs/api/browser/pause/)) maybe we can use this instead.